ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -3698,7 +3698,38 @@ tests:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-vsphere-ipi-ovn-dualstack
 - as: vsphere-ipi-ovn-dualstack-f28-ui
-  cron: 42 7 2 * *
+  cron: 32 9 29 * *
+  steps:
+    cluster_profile: vsphere
+    env:
+      E2E_RUN_TAGS: '@vsphere-ipi and @network-ovnkubernetes and not @fips and @console'
+      TAG_VERSION: '@4.14'
+    test:
+    - chain: openshift-e2e-test-qe-ui
```

Comment:
Missing install workflow?

Reply:
This is a separate job for ui only, I don't know why it needs update. UI separate jobs were added in https://github.com/openshift/release/pull/43482/files

Reply:
I think my comments is already resolved by @rbbratta